projects/iio_demo/src/main.c

```diff
@@ -192,7 +244,7 @@ int main(void)
 	iio_demo_out_init_par = (struct iio_demo_init_param) {
 		.name = demo_device_output,
 		.num_channels = 4,
-		.ddr_base_addr = DAC_DDR_BASEADDR,
+		.ddr_base_addr = OUTPUT_BASE_ADDRESS,
```

Comment:
if you make this change the demo won't work with xilinx anymore. Did you test this ?

Reply:
Forget this i just saw the define above. Maybe just use one define for Xilinx also not use another define for an existing one just change the name to OUTPUT_BASE_ADDRESS. Also try to separate the modifications in more commits it is hard to follow this change in just one commit named "Aducm support for iio_demo".

Reply:
done